libs/features/mas/commerce/checkout-link.md

```diff
@@ -38,6 +38,8 @@ See [MAS](mas.html#terminology) to learn more.
 | `data-entitlement`            | `entitlement` flag for client side interpretation                                                                                                                                                                                            | `false`       | `false`  |
 | `data-upgrade`                | `upgrade` flag for client side interpretation                                                                                                                                                                                                | `false`       | `false`  |
 | `data-modal`                  | `modal` flag for client side interpretation                                                                                                                                                                                                  | `false`       | `false`  |
+| `data-analytics-id`           | human-readable, non-translatable link id for analytics. Authored in Studio in Link Editor.                                                                                                                                                                                                  | `false`       | `false`  |
+| `daa-ll`                      | martech-compatible link id for analytics. Format: '${data-analytics-id}-${#}', where # is the position of the link within a card. E.g. : see-terms-1,  buy-now-2                                                                                                                                                                  | `false`       | `false`  |
```

Comment:
sorry to jump on this on your PR but we should probably somehow make a differentiation in documenting attributes that are inputs and those that are outputs. or "generated"? 
looks like spectrum is not making the difference, see `pending` in https://opensource.adobe.com/spectrum-web-components/components/picker/api/ but that does not mean it is right :)